base/exports.jl

```diff
@@ -1063,3 +1063,67 @@ export
     @view,
     @views,
     @static
+
+# TODO: use normal syntax once JuliaSyntax.jl becomes available at this point in bootstrapping
+eval(Expr(:public,
+    Symbol("@assume_effects"),
+    Symbol("@constprop"),
+    Symbol("@locals"),
+    Symbol("@propagate_inbounds"),
+    :AbstractLock,
+    :AsyncCondition,
+    :CFunction,
+    :Checked,
```

Comment:
Add `Filesystem`, `Sorting`, `Math`,`Orders`, and possibly others?

Reply:
`Filesystem`, `Sort`, and `Order`, yes. For `Math`, I think all public symobls in Math are already exported from Base so there is no need.